datamap/mv/core/src/test/scala/org/apache/carbondata/mv/rewrite/MVCreateTestCase.scala

```diff
@@ -1092,6 +1092,18 @@ class MVCreateTestCase extends QueryTest with BeforeAndAfterAll {
     assert(TestUtil.verifyMVDataMap(analyzed4, "constant_mv"))
   }
 
+  test("test mv query when the column names and table name same in join scenario") {
```

Comment:
Add some more tests with join , groupby with creating mv with both table duplicate columns , with and with out aliases

Reply:
done

Reply:
done